Tasks/DownloadPackageV0/task.json

```diff
@@ -10,7 +10,7 @@
     "version": {
         "Major": 0,
         "Minor": 161,
-        "Patch": 0
```

Comment:
I though you already made this 161.2? shouldn't this be more than that?

Reply:
This is DownloadPackageV0. The 161.2 was for DownloadPackageV1